### PR TITLE
Automate

### PR DESCRIPTION
**Pull request by kmcnellis**
This was my stuff - looks like it never got pull requested :/

### CODE REVIEW

File: github.js [hunk 1]
```diff
@@ -11,19 +11,22 @@ var webshotOptions = {
...
-var maxConcurrentThreads = 5;
+var maxConcurrentThreads = 10;
 var runningThreads = 0;
 var queuedThreads = [];
 
 function saveUserPageNoQueue(username, outputfile, callback){
     runningThreads ++;
     // Take screenshot of github page
-    webshot("http://www.github.com/" + username, outputfile, webshotOptions, function(err){
+        console.log("start: "+username);
```
Comment (seveibar) [inline github.js]: console log

Comment (seveibar) [inline github.js]: more descriptive

File: imggen.js [hunk 2]
```diff
@@ -9,35 +9,47 @@ try{
...
+var count = users.length;
 // Create page for every user
 for (var i = 0; i < users.length; i++){
     var username = users[i].github.login;
     (function(){
         var user = users[i];
-        github.saveUserPage(username,"./output/"+user.dirName+"/profile.jpg", function(){
-            console.log("Profile Generated for " + user.name);
-        });
+        // console.log(users[i].name, user.dirName);
```
Comment (seveibar) [inline imggen.js]: scary

File: pdfgen.js [hunk 3]
```diff
@@ -16,7 +17,33 @@ users.sort(function(a,b){
...
+//             }).reduce(function(previousValue, currentValue){
+//                 return previousValue + currentValue.name + " ("+currentValue.repositories + "),  ";
+//             }, "");
+//
+//         })(user)), 25,85)
+//
+//         .image("./output/" + user.dirName + "/profile.jpg",25,250,{
+//             fit: [540, 1000]
+//         });
+// }
```
Comment (seveibar) [inline pdfgen.js]: AHHHHH

Comment (seveibar) [inline pdfgen.js]: casper code?

Comment (kmcnellis) [inline pdfgen.js]: 👻

**Pull request merged**